src/Microsoft.Health.Dicom.Client/DicomWebClient.cs

```diff
@@ -352,6 +352,34 @@ public async Task<DicomWebResponse<ChangeFeedEntry>> GetChangeFeedLatest(string
                 });
         }
 
+        public async Task<DicomWebResponse> AddCustomTagAsync(IEnumerable<CustomTagEntry> customTagEntries, CancellationToken cancellationToken)
+        {
+            EnsureArg.IsNotNull(customTagEntries, nameof(customTagEntries));
```

Comment:
nit: I think we can change the formatting a bit:
1. Keep statements on one line as long as it isn't overly complex and it is shorter than like 100-120 characters
2. Logically separate blocks with newlines

Eg.
```csharp
EnsureArg.IsNotNull(customTagEntries, nameof(customTagEntries));

using var request = new HttpRequestMessage(HttpMethod.Post, BaseTagsUri)
{
    Content = new StringContent(JsonConvert.SerializeObject(customTagEntries)),
};

HttpResponseMessage response = await HttpClient.SendAsync(request, cancellationToken).ConfigureAwait(false);
await EnsureSuccessStatusCodeAsync(response).ConfigureAwait(false);
return new DicomWebResponse(response);
```